app/events/event.tsx

```diff
@@ -0,0 +1,30 @@
+
+
+import React from 'react';
+
+interface EventProps {
```

Comment:
should we add the date to the interface?

Reply:
^ to clarify, is the props.volunteers showing the number of volunteers in total for the event or the number of volunteers still needed?

Reply:
It was meant to just be the total number of volunteers needed (but I guess we could also make it decrement as more volunteers sign up?)

Reply:
Okay thank you! Sounds good just thinking about future iterations this looks solid to me.